scripts/utils/fcp_pop_helper.py

```diff
@@ -96,6 +97,7 @@ def train_agent_with_checkpoints(args, total_training_timesteps, ck_rate, seed,
         hidden_dim=h_dim,
         seed=seed,
         fcp_ck_rate=ck_rate,
+        curriculum=Curriculum(train_types=args.fcp_train_types, is_random=True)
```

Comment:
Recommend adding a comment explaining why we want `is_random=True` for generating the FCP population